AmplifyPlugins/Analytics/AWSPinpointAnalyticsPluginTests/AWSPinpointAnalyticsPluginConfigureTests.swift

```diff
@@ -59,4 +59,17 @@ class AWSPinpointAnalyticsPluginConfigureTests: AWSPinpointAnalyticsPluginTestBa
         }
     }
 
+    func testConfigureFailureForNilConfiguration() throws {
+        let plugin = AWSPinpointAnalyticsPlugin()
+        do {
+            try plugin.configure(using: nil)
+            XCTFail("Analytics configuration should not succeed")
+        } catch {
+            guard let apiError = error as? PluginError,
```

Comment:
I assume we're not really throwing an API error here, it's just a misnamed variable?